### PR TITLE
Document the New Features of GatorGrader in the README

### PR DESCRIPTION
**Pull request by corlettim**
This PR is for adding in the new features, that GatorGrader has adopted in, into the README. While most of the objects in the issue were tackled, as discussed in meetings, we felt that it was necessary to leave out explanations of checks since that would be explained in gatorgrader.org

### CODE REVIEW

**Comment by Michionlion** [comment]
@corlettim, as a remote contributor it would be a lot easier to help if you could link or name the relevant issues or discussions, if they are available! :)

File: README.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ to encounter:
 * `-h`
 * `--help`
 * `--exact`
-* `--advanced`
+* `--advanced` <br>
```
Comment (Michionlion) [inline README.md]: One should probably use a double line space here, or other markdown features to ensure a linebreak, rather than resorting to HTML

**Comment by schultzh** [comment]
Are we just waiting for the line endings issue to be resolved before this is merged? The domain name has been purchased, the site is live, and everything else seems to be in order.

**Comment by schultzh** [comment]
After re-acquainting myself with this issue, I recall that our stumbling block was various line ending formats from different operating systems. This can be seen through the Travis CI checks passing for MacOS but not Linux. A solution to this problem may be reconfiguring Git to handle line endings in a uniform way. [Please see this linked article.](https://docs.github.com/en/free-pro-team@latest/github/using-git/configuring-git-to-handle-line-endings) Any feedback would be appreciated, thanks!

File: README.md [hunk 2]
```diff
@@ -23,27 +23,39 @@ speed!
...
+to check programs and documentation for different languages:
+[Java](https://github.com/GatorEducator/java-assignment-starter-100-01),
+[Python](https://github.com/GatorEducator/python-assignment-starter-203-05), [LaTex](https://github.com/GatorEducator/latex-assignment-starter-100-01) and [HTML with CSS](https://github.com/GatorEducator/html-assignment-starter-302-03). These
 examples also show how to integrate GatorGrader with [GitHub
 Classroom](https://classroom.github.com/) and [Travis
 CI](https://travis-ci.com/). When you follow these examples, the Gradle plugin
 for GatorGrader will install it automatically when you run `gradle grade` in a
-terminal window.
+terminal window. Please be aware that these repositories are meant to have a majority
+of red checks. This is only suppose to be a “starter” and give an insight into what must be accomplished within the lab.
```
Comment (schultzh) [inline README.md:44]: "supposed" or "meant"

File: README.md [hunk 3]
```diff
@@ -23,27 +23,39 @@ speed!
...
-writing, then you can use GatorGrader's [Gradle
-plugin](https://github.com/GatorEducator/gatorgradle). The following starter
-repositories provide examples of how to configure GatorGrader to check programs
-and documentation for different languages:
-[Java](https://github.com/GatorEducator/java-starter),
-[LaTeX](https://github.com/GatorEducator/latex-assignment-starter), and [HTML
-with CSS](https://github.com/GatorEducator/html-css-assignment-starter). These
+* Starter Repositories
+An easy way to get started with GatorGrader is to check out our sample starter repositories.
+The following starter repositories provide examples of how to configure GatorGrader
```
Comment (schultzh) [inline README.md:35]: the language "how to configure GG to do [blank]" is a bit clunky
Could this be re-phrased so the word "to" is not repeated?

*same in the first sentence of solution repos

File: README.md [hunk 4]
```diff
@@ -23,27 +23,39 @@ speed!
...
+repositories. The following solution repositories provide examples of how to configure
+GatorGrader to check programs and documentation for different languages:
+[Java](https://github.com/GatorEducator/java-assignment-solution-100-01),
+[Python](https://github.com/GatorEducator/python-assignment-solution-203-05), [LaTex](https://github.com/GatorEducator/latex-assignment-solution-100-01) , and [HTML with CSS](https://github.com/GatorEducator/html-assignment-solution-302-03). These examples
+also show how to integrate GatorGrader with  [GitHub
+Classroom](https://classroom.github.com/) and [Travis
+CI](https://travis-ci.com/).  When you follow these examples, the Gradle plugin
+for GatorGrader will install it automatically when you run `gradle grade` in a
+terminal window. Please be aware that these repositories are meant to have a majority
+of green checks. This is suppose to show what must be accomplished within a lab/practical
```
Comment (schultzh) [inline README.md:56]: "supposed" or "meant"

File: README.md [hunk 5]
```diff
@@ -270,6 +276,31 @@ set by the [Proselint tool](http://proselint.com/). Since GatorGrader can run an
 arbitrary command and check its error code, it is also possible to integrate it
 with a wide variety of other linters, code formatters, and testing tools.
 
+Instructors may at times need to see a full list of checks to have a better understanding
+and therefore, we feel that it is important to know that there is an easy way to do this.
+This will be a command line and therefore, you can type
```
Comment (schultzh) [inline README.md:281]: Could we rephrase this sentence to make it more concise and avoid starting two sentence with "this" back to back?

**Review by schultzh — COMMENTED** [review]
I added some minor grammar/writing style comments on the changes, but overall this looks good! Thanks for all your hard work @corlettim. I'm excited to finally get this merged!

**Review by schultzh — APPROVED** [review]

**Review by Jordan-A — APPROVED** [review]
I reviewed the suggested changes for `README.md` and I agree that with the suggested writing adjustments it looks like #209 is ready to be merged. This PR will certainly improve the documentation of all of the ways to use GatorGrader.

**Review by donizk — APPROVED** [review]
changes approved, ready to merge

**Pull request merged**